x-pack/test/apm_api_integration/tests/metrics/serverless_overview.spec.ts

```diff
@@ -0,0 +1,406 @@
+          it('returns correct number of serverless functions', () => {
+            expect(
+              serverlessMetrics.serverlessFunctionsOverview.map((item) => {
+                return item.serverlessFunctionName;
+              })
+            ).to.eql(pythonServerlessFunctionNames);
+          });
+          it('returns correct values for all serverless functions', () => {
+            pythonServerlessFunctionNames.forEach((name) => {
+              const functionOverview = serverlessMetrics.serverlessFunctionsOverview.find(
+                (item) => item.serverlessFunctionName === name
+              );
+
+              expect(functionOverview?.serverlessDurationAvg).to.eql(FAAS_DURATION);
+              expect(functionOverview?.serverlessDurationAvg).to.eql(BILLED_DURATION_MS);
+              expect(functionOverview?.coldStartCount).to.eql(numberOfTransactionsCreated);
+              expect(functionOverview?.avgMemoryUsed).to.eql(expectedMemoryUsed);
+              expect(functionOverview?.memorySize).to.eql(MEMORY_TOTAL);
+            });
+          });
+        });
+
+        describe('serverless active instances overview', () => {
+          it('returns two serverless functions from the same instace', () => {
+            expect(
+              serverlessMetrics.serverlessActiveInstancesOverview.map((item) => {
+                return {
+                  serverlessFunctionName: item.serverlessFunctionName,
+                  instance: item.activeInstanceName,
+                };
+              })
+            ).to.eql(
+              pythonServerlessFunctionNames.map((name) => {
+                return { serverlessFunctionName: name, instance: PYTHON_INSTANCE_NAME };
+              })
+            );
+          });
+
+          it('returns correct values for all serverless functions', () => {
+            pythonServerlessFunctionNames.forEach((name) => {
+              const activeInstanceOverview =
+                serverlessMetrics.serverlessActiveInstancesOverview.find(
+                  (item) => item.serverlessFunctionName === name
+                );
+
+              expect(activeInstanceOverview?.serverlessDurationAvg).to.eql(FAAS_DURATION);
+              expect(activeInstanceOverview?.serverlessDurationAvg).to.eql(BILLED_DURATION_MS);
+              expect(activeInstanceOverview?.avgMemoryUsed).to.eql(expectedMemoryUsed);
+              expect(activeInstanceOverview?.memorySize).to.eql(MEMORY_TOTAL);
+            });
+          });
+
+          describe('timeseries', () => {
+            it('returns correct mean values', () => {
+              pythonServerlessFunctionNames.forEach((name) => {
+                const activeInstanceOverview =
+                  serverlessMetrics.serverlessActiveInstancesOverview.find(
+                    (item) => item.serverlessFunctionName === name
+                  );
+
+                function removeNullAndZeroValue(coordinate: Coordinate) {
```

Comment:
Is this needed? If so, an alternative name might make more sense such (e.g. `isNullOrZeroCoordinate`) as it doesn't remove anything from the param

Reply:
yeah, it's needed otherewise the mean value doesn't match the value I want because of the bucket size.